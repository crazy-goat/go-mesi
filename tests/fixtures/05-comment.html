<!DOCTYPE html>
<html>
<head>
    <title>Standard esi:remove tests</title>
</head>
<body>
This should be empty: [<esi:comment text="some comment here" />]
This should be empty: [<!--esi <esi:comment text="some comment here" />-->]
</body>
</html>
