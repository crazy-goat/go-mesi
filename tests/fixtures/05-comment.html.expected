<!DOCTYPE html>
<html>
<head>
    <title>Standard esi:remove tests</title>
</head>
<body>
This should be empty: []
This should be empty: []
</body>
</html>
